Week8-2/NetworkManager/WebService.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Webservice.swift
+//  Week8-2
+//
+//  Created by Mehmet Salih ÇELİK on 22.11.2021.
+//
+
+import Foundation
+import Alamofire
+
+class WebService {
+    
+    typealias CompletionHandler = (_ success: [Photo] ) -> Void
```

Comment:
Result success ve error blogu kullanılması daha doğru olur hata durumları için

Reply:
fixed